.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,38 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ develop, release/** ]
+  pull_request:
+    branches: [ develop ]
+  schedule:
+    - cron: '45 17 * * 2'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript', 'python' ]
```

Comment:
typescript ?

Reply:
It looks like this is transparent for CodeQL...

Reply:
just saying ;-)
I've read in the doc that typescript is supported

Reply:
Do both in same time :)